tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/passes/BoundedCheckerPassImpl.scala

```diff
@@ -73,10 +73,11 @@ class BoundedCheckerPassImpl @Inject() (val options: PassOptions,
         val random = tuning.getOrElse("search.randomDfs", "")
         new DfsStrategy(input, stepsBound, random.toLowerCase.equals("true"))
       }
+    val tlc = options.getOrElse("checker", "tlc", false)
 
     val checker: Checker =
         new ModelChecker(typeFinder, hintsStore, changeListener, exprGradeStore, sourceStore,
-          input, strategy, tuning, debug, profile, checkRuntime)
+          input, strategy, tuning, debug, profile, checkRuntime, tlc)
```

Comment:
if we dumped the counterexamples in all formats, we would not need the extra parameter